wiki/Replay/en.md

```diff
@@ -0,0 +1,285 @@
+
+A white, hovering text from right-to-left will be shown on top but below the **UNRANKED** notice when watching a replay.
+
+The format is shown below unless stated otherwise:-
+
+```
+Format : REPLAY MODE - Watching {AccountName} play {ArtistName} - {BeatmapsetName} [{Difficulty}]
+Example: REPLAY MODE - Watching osu! play Peter Lambert - osu!tutorial [Gameplay Basics]
+```
+
+### Local (Solo)
+
+![A local replay][Replay Solo]
+
+_A typical local replay_
+
+Solo local plays regardless of connection status or logged-in account name.
+As long as the difficulty was passed, the replay will be generated internally and can be exported to `Replays` folder.
+
+At the backend, one pair of files will be created in the hidden folder, `Data\r`: `.osr` (_osu replay_; the replay file), and `.osg` (accuracy and combo in real-time; used for spectators' replay only) with same encrypted filename.
+It is highly suggested to **not rename the encrypted filename**, and make use of the export `F2` key instead.
+
+Deleting the `.osg` file has no effect (it is a leftover file for spectator's replay; can be safely removed).
+Deleting the `.osr` file will make the replay _lost forever_ (since the replay data is missing and nothing to export).
+Alternatively, deleting the `scores.db` (contains local leaderboard scores and pointers to encrypted replays) _will make all (not exported) replays and local leaderboard scores lost forever_.
+
+To set a name when not logged-in (_Guest_ account), scroll down the result screen to access the _online results screen_ directly after passing the difficulty and type in the name under _Guest player name_.
+In this screen, an alternative to using the export `F2` key is to enable the `Save replay to Replays folder` button located at the upper-right.
+
+Return to the _Song Selection_ and changes will be updated for this replay (if no name was given, it will simply be blank).
+
+### Failed
+
+![Comparison of game over screen between normal and failed replay][Replay failed]
+
+_Left is the typical game over screen. Right is showing the end of a failed replay._
+
+To access this screen, fail the map by draining the healthbar completely, or fail to fill pass 50% of the healthbar before the end of the difficulty in osu!taiko case.
+In the game over screen, press the `F1` key to replay the currently failed play.
+The replay ends when the screen darkens, with osu!logo in the middle, rather than the game over screen.
+
+Due to the non-standard replay screen, use the `Esc` key to return to Song Selection.
+Retrying the play (as in, actual play) of the beatmap in this state is not possible.
+
+It currently works in Solo mode only when given the game over screen, and does not work if [No Fail(NF)][No Fail wikilink]/ [Relax(RL)][Relax wikilink]/[Auto Pilot(AP)][Auto Pilot wikilink] (impossible to fail) or [Perfect(PF)][Perfect wikilink] (forcefully quick-retry instead) game modifier was used.
```

Comment:
Viewing a failed play does not work with the [No Fail(NF)][No Fail wikilink]/ [Relax(RL)][Relax wikilink]/[Auto Pilot(AP)][Auto Pilot wikilink] game modifiers because it is impossible to fail. Using the [Perfect(PF)][Perfect wikilink] game modifier would forcefully quick-retry instead of failing the player.